.travis.yml

```diff
@@ -23,9 +23,9 @@ before_script:
 
 script:
    #run template app generation
-   # - travis_wait 30 set -o pipefail && node external/shared/node/test_force.js --os=ios --test=native
-   # - travis_wait 30 set -o pipefail && node external/shared/node/test_force.js --os=ios --test=native_swift
-   # - travis_wait 30 set -o pipefail && node external/shared/node/test_force.js --os=ios --test=react_native
+   - travis_wait 30 set -o pipefail && node external/shared/node/test_force.js --os=ios --test=native
+   - travis_wait 30 set -o pipefail && node external/shared/node/test_force.js --os=ios --test=native_swift
+   - travis_wait 30 set -o pipefail && node external/shared/node/test_force.js --os=ios --test=react_native
```

Comment:
not sure why failed in run 803 on travis, but seems all passing in recent runs, so add back